### PR TITLE
19549 - fix legal-api ci errors

### PR DESCRIPTION
**Pull request by tshyun24**
*Issue #:* /bcgov/entity#19549

*Description of changes:*
**Talked with Argus, the testing will be fixed in the future ticket**

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

**Review by leodube-aot — APPROVED** [review]
LGTM

File: legal-api/pyproject.toml [hunk 1]
```diff
@@ -79,19 +85,99 @@ docstring-min-length=10
...
+exclude-protected = ["_asdict", "_fields", "_replace", "_source", "_make", "os._exit"]
+valid-classmethod-first-arg = ["cls"]
+valid-metaclass-classmethod-first-arg = ["mcs"]
+
+[tool.pytest.ini_options]
+minversion = "2.0"
+testpaths = [
+   "tests",
+]
+addopts = "--verbose --strict -p no:warnings --cov=. --cov-report html:htmlcov --cov-report xml:coverage.xml"
```
Comment (argush3) [inline legal-api/pyproject.toml:143]: update `--cov=.` to `--cov=src`

File: legal-api/pyproject.toml [hunk 2]
```diff
@@ -79,19 +85,99 @@ docstring-min-length=10
...
+markers = [
+   "slow",
+   "serial",
+]
+
+[tool.coverage.run]
+branch = true
+source = [
+   ".",
+]
```
Comment (argush3) [inline legal-api/pyproject.toml:164]: Update to 

```python
source = [
   "src/legal-api",
]
```

File: legal-api/pyproject.toml [hunk 3]
```diff
@@ -79,19 +85,99 @@ docstring-min-length=10
...
+omit = []
```
Comment (argush3) [inline legal-api/pyproject.toml:165]: update to

```python
omit = [
    "wsgi.py",
    "gunicorn_config.py"
]
```

**Comment by argush3** [comment]
Looks good, just a few things for the code coverage config in the `pyproject.toml`

**Pull request merged**